test/k8s-integration/main.go

```diff
@@ -106,6 +108,20 @@ func main() {
 	if !*inProw && *doDriverBuild {
 		ensureVariable(stagingImage, true, "staging-image is a required flag, please specify the name of image to stage to")
 	}
+
+	if *useStagingDriver {
+		ensureVariableVal(deploymentStrat, "gke", "'deployment-strategy' must be GKE for using managed driver")
+		ensureFlag(doDriverBuild, false, "'do-driver-build' must be false when using GKE managed driver")
+		ensureFlag(teardownDriver, false, "'teardown-driver' must be false when using GKE managed driver")
+		ensureVariable(stagingImage, false, "'staging-image' must not be set when using GKE managed driver")
+		ensureVariable(deployOverlayName, false, "'deploy-overlay-name' must not be set when using GKE managed driver")
+	}
+
+	if !*useStagingDriver {
+		ensureVariable(deployOverlayName, true, "deploy-overlay-name is a required flag")
+		ensureVariable(saFile, true, "service-account-file is a required flag")
```

Comment:
If deployOverlayName != 'dev', only then saFile is needed.

Reply:
done